spec/features/01_can_create_account_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+feature 'can create a new user account' do
+  scenario 'visits sign up page' do
+    visit '/users/sign_up'
+    expect(page).to have_content 'Sign up'
+  end
+
+  scenario 'creates an account' do
```

Comment:
It's a good idea to break up your tests a little bit, especially feature tests like this that are a bit longer. I like to break my down into two/ three sections generally (setup, action, and expectation); you can also break down these feature tests that have a lot of 'action' portion further. If I were to write this test (and you may write it differently, this is preference keep in mind), I would have put a line after saying where to visit, after filling out the form completely (including submitting maybe), and then having the expectation on its own. Pick a style of how you're going to break up your tests and keep it consistent throughout your codebase